beta/src/pages/learn/index.md

```diff
@@ -379,13 +379,37 @@ button {
 
 </Sandpack>
 
+Notice how each button "remembers" its own `count` state and doesn't affect other buttons.
+
+## Using Hooks {/*using-hooks*/}
+
 Functions starting with `use` are called Hooks. `useState` is a built-in Hook provided by React. You can find other built-in Hooks in the [React API reference](/reference). You can also write your own Hooks by combining the existing ones.
 
 Hooks are more restrictive than regular functions. You can only call Hooks *at the top level* of your components (or other Hooks). If you want to `useState` in a condition or a loop, extract a new component and put it there.
 
 ## Sharing data between components {/*sharing-data-between-components*/}
 
-In the above example, each counter is independent. If you want them to display the same value and update together, *move the state up* to their closest common ancestor. First, move state from `MyButton` into `MyApp`:
+In the previous example, each button had its own counter. Imagine the structure of that example:
+
+```js {2-4}
+- MyApp
+  - MyButton (count: 3)
+  - MyButton (count: 1)
+  - MyButton (count: 2)
+```
+
+If you want all buttons to display the same `count` and *update together* on click, you need to move the state from the individual buttons "upwards" to the closest component containing all of them. In this example, it is `MyApp`:
```

Comment:
I really like this paragraph because it provides a further explanation.

Reply:
>Because it feels more as a "btw" given that later we build the opposite.

Not sure I'm following. That _is_ what we build later.

Reply:
riiiight.
( jetlag brain 🤦‍♀️ )

Reply:
I pushed a hopefully clearer explanation. The ordering was correct — the thing we're building next is the second case. should be a bit clearer now.